packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -591,13 +591,13 @@ async function handler(req: CustomRequest) {
     }
 
     // Posible to refactor TODO:
-    const paymentApp = await appStore[paymentAppCredential?.app?.dirName as keyof typeof appStore];
+    const paymentApp = await appStore[paymentAppCredential?.app?.dirName as keyof typeof appStore]();
     if (!(paymentApp && "lib" in paymentApp && "PaymentService" in paymentApp.lib)) {
       console.warn(`payment App service of type ${paymentApp} is not implemented`);
       return null;
     }
 
-    const PaymentService = paymentApp.lib.PaymentService;
+    const PaymentService = paymentApp.lib.PaymentService as any;
```

Comment:
Do we really need this type assertion?

Reply:
Yep, I think we must create a generic PaymentService type at this point  the PaymentService attributes is unknown, thoughts?

Reply:
Yeah, I think that would be the right approach. We do something similar for  Calendar.
<img width="889" alt="Screenshot 2023-04-27 at 3 14 46 PM" src="https://user-images.githubusercontent.com/1780212/234825257-33dbbbf9-77d6-4650-9e15-9fc9324b21dd.png">

Reply:
@roae I don't understand why TypeScript isn't able to infer types for PaymentService, it is still able to do it for CalendarService. For now, I have asserted the type on the instance of PaymentService so that we can call it's methods safely.